crates/compiler/fmt/src/test_helpers.rs

```diff
@@ -1,77 +1,82 @@
-use bumpalo::Bump;
-use roc_test_utils::assert_multiline_str_eq;
 
-use crate::{
-    annotation::{Formattable, Newlines, Parens},
-    Buf,
-};
 
-/// Parse and re-format the given input, and pass the output to `check_formatting`
-/// for verification.  The expectation is that `check_formatting` assert the result matches
-/// expectations (or, overwrite the expectation based on a command-line flag)
-/// Optionally, based on the value of `check_idempotency`, also verify that the formatting
-/// is idempotent - that if we reformat the output, we get the same result.
-pub fn expr_formats(input: &str, check_formatting: impl Fn(&str), check_idempotency: bool) {
-    let arena = Bump::new();
-    let input = input.trim();
+#[cfg(test)]
```

Comment:
Making this `#[cfg(test)]` will break the fuzzing. FWIW, parse has its own `test_helpers.rs` that's not cfg'd like this.

What's the purpose of this change?

Reply:
It's so that `roc_test_utils` can be a dev-dependency, which it should always be. Trying to include it as a regular dependency caused problems for the wasm-repl. The `[target.'cfg(not(target_family = "wasm"))'.dependencies]` I added will also fix this. But I want to prevent unnecessary compilation and problems like this that are caused by unnecessary dependencies that are never used in the roc executable.

How can I run the fuzzing?

Reply:
Ah, interesting.

Run fmt fuzzing like this:
```
cd crates/compiler/fmt
cargo +nightly fuzz run -j4 fuzz_expr -- -dict=fuzz/dict.txt
```

(also make sure you have a semi-recent nightly installed)